ros/src/twist_controller/launch/dbw_sim.launch

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <launch>
-    <node pkg="twist_controller" type="dbw_node.py" name="dbw_node">
+    <node pkg="twist_controller" type="dbw_node.py" name="dbw_node" output="screen">
```

Comment:
what does this do? did you change manually?

Reply:
It allows you to `print()` and see the result in the command line. I'll remove it, we might want to use `rospy.loginfo`

Reply:
good to know :-)